Jenkinsfile.github

```
@@ -0,0 +1,40 @@
+@Library('csm-shared-library') _
+
+pipeline {
+    agent {
+        label "metal-gcp-builder"
+    }
+
+    options {
+        buildDiscarder(logRotator(numToKeepStr: "10"))
+        timestamps()
+    }
+
+    environment {
+        NAME = "cray-vpa"
+        DESCRIPTION = "Deploys the vpa policy to CSM systems"
+        IS_STABLE = getBuildIsStable()
+        CHART_NAME_1 = "cray-vpa"
+        CHART_VERSION_1 = getChartVersion(name: env.CHART_NAME_1, isStable: env.IS_STABLE)
```

Dont need the "_1" theres only one chart
```suggestion
        CHART_VERSION = getChartVersion(name: env.CHART_NAME, isStable: env.IS_STABLE)
```